test/e2e/util/config.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+import (
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+const (
+	DefaultClusterName   = "e2e-test"
+	DefaultNamespace     = "kvrocks"
+	DefaultKruiseVersion = "1.4.0"
+	DefaultMinikubeShell = "start_minikube_cluster.sh"
+)
+
+type Config struct {
+	KruiseVersion string `yaml:"kruiseVersion"`
+	ClusterName   string `yaml:"clusterName"`
+	Namespace     string `yaml:"namespace"`
+	KubeConfig    string `yaml:"kubeConfig"`
```

Comment:
Do all variables declared in a struct need to be exported?

Reply:
That's helpful. I have already hidden some variables.

Reply:
I have fixed the bug.